web_src/css/repo/home-file-list.css

```diff
@@ -22,39 +33,63 @@
   display: contents;
 }
 
-#repo-files-table .repo-file-item:hover > .repo-file-cell {
+#repo-files-table .repo-file-item:hover > .repo-file-cell,
+#repo-files-table .parent-link:hover {
   background: var(--color-hover);
 }
 
 #repo-files-table .repo-file-line,
 #repo-files-table .repo-file-cell {
-  border-top: 1px solid var(--color-light-border);
-  padding: 6px 10px;
+  border-top: .5px solid var(--color-light-border);
+  border-bottom: .5px solid var(--color-light-border);
+  padding: 8px 10px;
 }
 
 #repo-files-table .repo-file-line:first-child {
   border-top: none;
+  border-radius: var(--border-radius) var(--border-radius) 0 0;
+}
+
+#repo-files-table .repo-file-item:last-child .repo-file-line,
+#repo-files-table .repo-file-item:last-child .repo-file-cell {
+  border-bottom: none;
+}
+
+#repo-files-table .repo-file-item:last-child {
+  border-bottom: none;
+}
```

Comment:
IIRC `repo-file-item` doesn't have border

Reply:
Hmm yes this may be superflous.